src/ViewHelpers/DebugViewHelper.php

```diff
@@ -111,11 +131,16 @@ protected static function dumpVariable($variable, $html, $level, $levels)
                 }
             }
         } else {
-            if (is_scalar($variable) || is_null($variable)) {
+            if (is_scalar($variable) || $variable === null) {
                 $string = sprintf(
                     '<code>%s = %s</code>',
                     $typeLabel,
-                    htmlspecialchars(var_export($variable, true), ENT_COMPAT, 'UTF-8', false)
+                    htmlspecialchars(
```

Comment:
I don't mean to nit-pick here, but the whitespace formatting is now applied inconsistently - there are several other places where a similar breaking isn't done even though there are nested function calls.

Rule of thumb is: if you have to use three or more functions, always break them out. If the line remains shorter than the ~120 char limit, usually there's no need to break it.